source/class/qx/ui/table/pane/FocusIndicator.js

```diff
@@ -125,7 +125,8 @@ qx.Class.define("qx.ui.table.pane.FocusIndicator", {
             }
           }
           var userHeight = rowHeight + (wl + wr - 2);
-          var userTop = (row - firstRow) * rowHeight - (wr - 1);
+          var renderedRowHeight = this.__scroller.getTablePane().getRenderedRowHeight();
+          var userTop = Math.floor((row - firstRow) * renderedRowHeight) - (wr - 1);
```

Comment:
It looks like you are correct that the original code, and now your new code too, uses decoration widths from left and right rather than top and bottom as they should. To confirm that, maybe try modifying the decorator to have drastically different left and right widths than the top and bottom widths, and see how it looks. My suspicion is that it has worked to date only because those widths are all the same.

Reply:
So, as suspected this did not work well when the widths were changed in the decorator. I have added a commit amending the errors.